docs/admin/backup/restore.md

```diff
@@ -38,7 +38,7 @@ In the event of disaster, you can restore from a backup by doing the following:
     kind: Restore
     metadata:
       name: <restore-name>
-      namespace: kcm-system
+      namespace: <your-namespace>

```

Comment:
Please leave as it is rn or change to `<kcm-system-namespace>` (or similar), the namespace must be the system namespace, in which `kcm` has been installed in the first place. Random namespace will definitely not work

This comment applies elsewhere the `<your-namespace>` is mentioned for `backups` or `restores` (both velero CRs, not `managementbackups` of k0rdent)